aspnetcore/blazor/security/webassembly/hosted-with-azure-active-directory-b2c.md

```diff
@@ -317,6 +317,7 @@ All scopes must belong to the same app per Azure Active Directory rules. Additio
 builder.Services.AddMsalAuthentication(options =>
 {
     ...
+    options.ProviderOptions.ResponseType = "id_token token";
     options.ProviderOptions.DefaultAccessTokenScopes.Add("{SCOPE URI}");
```

Comment:
The implicit flow is no longer recommended and I don't think this should be used in the docs. Using code flow with PKCE or a confidential client would be better?

Greetings Damien

Reply:
In that case, should I document to use code flow instead of ``id_token token`` ?

Reply:
Here is a new version of the PR